test_utils/factories.py

```diff
@@ -71,6 +78,24 @@ class Meta(object):
     user_id = factory.LazyAttribute(lambda x: FAKER.pyint())
 
 
+class UserDataSharingConsentAuditFactory(factory.django.DjangoModelFactory):
+    """
+    UserDataSharingConsentAuditFactory.
+
+    Creates an instance of UserDataSharingConsentAudit with minimal boilerplate.
+    """
+
+    class Meta(object):
+        """
+        Meta for UserDataSharingConsentAuditFactory.
```

Comment:
I would certainly be a fan of that. Can you point me in the right direction? I found a way to restrict linting based on block line length, but not on class name.